IdentityServer/IdentityServer/Program.cs

```diff
@@ -0,0 +1,30 @@
+using IdentityServer;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddControllers();
+
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+builder.Services.AddIdentityServer()
+    .AddInMemoryIdentityResources(Configuration.GetIdentityResources())
+    .AddInMemoryClients(Configuration.GetClients())
+    .AddInMemoryApiResources(Configuration.GetApiResources())
```

Comment:
Leave it by now, you'll need to store configuration such as clients, resources etc in database